app/extensions/brave/brave-default.js

```diff
@@ -604,14 +604,21 @@ if (typeof KeyEvent === 'undefined') {
       }
 
       const selection = getSelection()
-      let suggestions = []
-      let isMisspelled = false
-      if (selection.length > 0 && !hasWhitespace(selection)) {
-        // This is not very taxing, it only happens once on right click and only
-        // if it is on one word, and the check and result set are returned very fast.
-        const info = ipcRenderer.sendSync('get-misspelling-info', selection)
-        suggestions = info.suggestions
-        isMisspelled = info.isMisspelled
+      let spellcheck = null
+      if (e.target.spellcheck && !e.target.readonly && !e.target.disabled) {
```

Comment:
I think this should be e.target.spellcheck !== 'false' but otherwise this looks good. I'll merge in and change in a followup. Thanks!